src/parser/expression.js

```diff
@@ -385,6 +385,16 @@ pp.parseExprAtom = function (refShorthandDefaultPos) {
       }
       return this.finishNode(node, "Super");
 
+    case tt._import:
+      if (!this.hasPlugin("importFunctions")) this.unexpected();
```

Comment:
I agree that would be nice. I did this because it seems like this is how the rest of the plugins handle it.